src/epoch.rs

```diff
@@ -332,6 +332,35 @@ impl Epoch {
         }) + Unit::Second * SECONDS_GPS_TAI_OFFSET
     }
 
+    /// Initialize an Epoch from the provided UNIX second timestamp since UTC midnight 1970 January 01.
+    pub fn from_unix_seconds(seconds: f64) -> Self {
+        let mut e = Self::from_tai_duration(
```

Comment:
I think this could be simplified as
```suggestion
        Self::from_utc_seconds(seconds + UNIX_OFFSET_UTC_SECONDS)
```

Reply:
After your suggestion about using the `from_gregorian_utc_at_midnight` I found I could just get rid of the `UNIX_OFFSET_UTC_SECONDS` const entirely. Making an epoch from gregorian UTC then converting to a duration is equivalent and less numbers to memorize.